IETF.js

```diff
@@ -0,0 +1,217 @@
+		var tmpDate = ZU.xpathText(metadataDoc, "//meta[@name='DC.Date.Issued']/@content");
+		newItem.date = ZU.strToISO(tmpDate);
+
+		// Adding the attachment
+		newItem.attachments.push({
+			title: "Snapshot",
+			mimeType: "text/html",
+			url: targetUri
+		});
+	
+		newItem.complete();
+	});
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://tools.ietf.org/html/rfc2474",
+		"items": [
+			{
+				"itemType": "report",
+				"title": "Definition of the Differentiated Services Field (DS Field) in the IPv4 and IPv6 Headers",
+				"creators": [
+					{
+						"lastName": "Nichols",
+						"firstName": "Kathleen",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Black",
+						"firstName": "David L.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Blake",
+						"firstName": "Steven",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Baker",
+						"firstName": "Fred",
+						"creatorType": "author"
+					}
+				],
+				"date": "1998-12",
+				"abstractNote": "This document defines the IP header field, called the DS (for differentiated services) field.",
+				"institution": "IETF",
+				"libraryCatalog": "IETF",
+				"reportNumber": "RFC 2474",
+				"reportType": "STANDARDS-TRACK",
+				"url": "https://tools.ietf.org/html/rfc2474",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://tools.ietf.org/html/rfc2822",
+		"items": [
+			{
+				"itemType": "report",
+				"title": "Internet Message Format",
+				"creators": [
+					{
+						"firstName": "Peter W.",
+						"lastName": "Resnick",
+						"creatorType": "author"
+					}
+				],
+				"date": "2001-04",
+				"abstractNote": "This document specifies a syntax for text messages that are sent between computer users, within the framework of \"electronic mail\"\nmessages.",
+				"institution": "IETF",
+				"libraryCatalog": "IETF",
+				"reportNumber": "RFC 2822",
+				"reportType": "STANDARDS-TRACK",
+				"url": "https://tools.ietf.org/html/rfc2822",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://tools.ietf.org/rfc/rfc2822.txt",
+		"items": [
+			{
+				"itemType": "report",
+				"title": "Internet Message Format",
+				"creators": [
+					{
+						"firstName": "Peter W.",
+						"lastName": "Resnick",
+						"creatorType": "author"
+					}
+				],
+				"date": "2001-04",
+				"abstractNote": "This document specifies a syntax for text messages that are sent between computer users, within the framework of \"electronic mail\" messages.",
+				"institution": "IETF",
+				"libraryCatalog": "IETF",
+				"reportNumber": "RFC 2822",
+				"reportType": "STANDARDS-TRACK",
```

Comment:
We should normalize these to, e.g., "Standards Track" (as shown in the report itself).